src/dispatcher/index.ts

```diff
@@ -100,35 +105,33 @@ const createDispatcher = () => {
   })();
 
   return {
-    _setLang: (l: string) => (lang = l),
+    _setLang: (l: string) => {
+      lang = l;
+    },
     _addToQueue: (type: HeadType, payload: MetaPayload | string): void => {
       processQueue();
 
-      switch (type) {
-        case 'title':
-          titleQueue.splice(currentTitleIndex++, 0, payload as string);
-          break;
-
-        case 'titleTemplate':
-          titleTemplateQueue.splice(
-            currentTitleTemplateIndex++,
-            0,
-            payload as string
-          );
-          break;
-
-        case 'meta':
-          metaQueue.splice(currentMetaIndex++, 0, payload as MetaPayload);
-          break;
-
-        default:
-          linkQueue.push(payload);
+      if (type === TITLE) {
```

Comment:
Hope you don't mind me reverting this, it added a lot of size :/

Reply:
I don't. Thought it would be more readable :)